requirements.txt

```diff
@@ -1,61 +1,64 @@
 #
 # This file is autogenerated by pip-compile
-# Make changes in requirements.in, then run this to update:
+# To update, run:
 #
-#    pip-compile requirements.in
+#    pip-compile --output-file requirements.txt requirements.in
 #
 argparse==1.4.0           # via ofs
 Babel==2.3.4
-Beaker==1.8.0
+Beaker==1.8.0             # via pylons
 bleach==1.4.3
+click==6.6                # via rq
 decorator==4.0.6          # via pylons, sqlalchemy-migrate
 fanstatic==0.12
 Flask==0.10.1
-FormEncode==1.3.0
+FormEncode==1.3.0         # via pylons
+funcsigs==1.0.2           # via beaker
 html5lib==0.9999999       # via bleach
 itsdangerous==0.24        # via flask
-Jinja2==2.8
-Mako==1.0.4
+Jinja2==2.8               # via flask
+Mako==1.0.4               # via pylons
 Markdown==2.6.6
-MarkupSafe==0.23
+MarkupSafe==0.23          # via jinja2, mako, webhelpers
 nose==1.3.7               # via pylons
 ofs==0.4.2
 ordereddict==1.1
 Pairtree==0.7.1-T
 passlib==1.6.5
 paste==1.7.5.1
-PasteDeploy==1.5.2
-PasteScript==2.0.2
-pbr==0.11.1               # via sqlalchemy-migrate
+PasteDeploy==1.5.2        # via pastescript, pylons
+PasteScript==2.0.2        # via pylons
+pbr==1.10.0               # via sqlalchemy-migrate
 psycopg2==2.4.5
-pysolr==3.5.0
-Pygments==2.1.3
+Pygments==2.1.3           # via weberror
 Pylons==0.9.7
+pysolr==3.5.0
 python-dateutil==1.5
 pytz==2016.4
 pyutilib.component.core==4.6.4
+redis==2.10.5             # via rq
 repoze.lru==0.6           # via routes
 repoze.who-friendlyform==1.0.8
 repoze.who==2.3
 requests==2.10.0
-Routes==1.13
+Routes==1.13              # via pylons
+rq==0.6.0
 simplejson==3.8.2         # via pylons
-six==1.10.0               # via bleach, html5lib, pastescript, sqlalchemy-migrate
+six==1.10.0               # via bleach, html5lib, pastescript, pyutilib.component.core, sqlalchemy-migrate
 sqlalchemy-migrate==0.10.0
-SQLAlchemy==0.9.6
+SQLAlchemy==0.9.6         # via sqlalchemy-migrate
 sqlparse==0.1.19
-Tempita==0.5.2
+Tempita==0.5.2            # via pylons, sqlalchemy-migrate, weberror
 tzlocal==1.2.2
 unicodecsv==0.14.1
 vdm==0.13
-WebError==0.13.1
-WebHelpers==1.3
-WebOb==1.0.8
-WebTest==1.4.3
-Werkzeug==0.11.10
+WebError==0.13.1          # via pylons
+WebHelpers==1.3           # via pylons
+WebOb==1.0.8              # via fanstatic, pylons, repoze.who, repoze.who-friendlyform, weberror, webtest
+WebTest==1.4.3            # via pylons
+Werkzeug==0.11.10         # via flask
```

Comment:
Not related to this PR, but there has to be a better way of tracking dependency sources? Any ideas?